### PR TITLE
Add IET and IMechE Sponsor logos

### PR DESCRIPTION
**Pull request by antoinepetty**
Pending
- Sending a render to them for approval
- Gaining their approval on this render

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -86,6 +86,15 @@ sponsors:
...
     image: /images/sponsors/itdev.png
 
+  - name: Institution of Engineering and Technology 
+    link: https://www.theiet.org/
+    image: /images/sponsors/iet.png
+
+  - name: Institution of Mechanical Engineers
+    link: https://www.imeche.org/
+    image: /images/sponsors/imeche.png
+
```
Comment (PeterJCLaw) [inline _config.yml:96]: In this order the balance of the page looks a bit odd:
image

Could we flip the order so the IET are on their own on the last line?
image

**Comment by sedders123** [comment]
With latest commit sponsor section looks like
image

File: _config.yml [hunk 2]
```diff
@@ -69,30 +69,37 @@ sponsors:
...
 
   - name: Systemetric - Hills Road Robotics
     link: https://systemetric.hr-robocon.org/
     image: /images/sponsors/systemetric-hills-road.svg
+    logo_width: default
 
   - name: ITDev
     link: https://www.itdev.co.uk/
     image: /images/sponsors/itdev.png
+    logo_width: 200px
```
Comment (antoinepetty) [inline _config.yml:92]: how does this scale with mobile etc?

Comment (sedders123) [reply _config.yml:92]: image

**Review by sedders123 — APPROVED** [review]
We have received approval of these logo placements from the IET and IMechE in an email sent to pr@ and fundraising@

**Pull request merged**